### ISSUE TITLE
RM slide breaks if calibration is not done first

### ISSUE THREAD

**Issue by mbsherma** [dsa110/dsa110-issues#306]
A "previous stages required" message should be displayed, but instead the screen takes forever to load nothing.

**Comment by mbsherma** [comment]
This is likely an issue with the initialization of the RM spectrum or calibrated spectra to all nan at native resolution. You can resolve by simply not plotting if RM synthesis hasn't been run yet.

**Comment by mbsherma** [comment]
Resolved in commit https://github.com/dsa110/dsa110-pol/commit/cc41ab34214f6f0db9952941e136ac6d013473c6 ; this was actually due to the ionospheric RM calculation taking too long. I've deferred this to when the user clicks the 'Get RMion' button, but will see if I can speed this up further (seems to be an issue with how the new ion RM module `https://github.com/lofar-astron/RMextract` queries data. To be safe, plotting of 1D synthesis results is deferred to after RM synthesis has been run, and labels have been added for both 1D and 2D synthesis to prevent confusion.
